test/e2e/oauth2-authorization-code.js

```diff
@@ -35,7 +35,7 @@ describe('oauth2 authorization code grant type', () => {
           path: '/not-found'
         }
       },
-      policies: ['oauth2', 'proxy'],
+      policies: ['cors', 'jwt', 'basic-auth', 'oauth2', 'proxy'],
```

Comment:
These policies do not seem to be used in this test. Why did you need to load them?

Reply:
The tests were failing
Logs:
```
  606 passing (2m)
  2 pending
  2 failing

  1) oauth2 authorization code grant type
       "before all" hook:
     Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/ravi/oss/express-gateway/test/e2e/oauth2-authorization-code.js)
  

  2) oauth2 authorization code grant type
       "after all" hook:
     TypeError: Cannot read property 'kill' of null
      at Context.done (test/e2e/oauth2-authorization-code.js:131:20)
```
With `LOG_LEVEL=debug`
```
  5) oauth2 authorization code grant type
       "before all" hook:
     Error: connect ECONNREFUSED 127.0.0.1:33635
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1097:14)

  6) oauth2 authorization code grant type
       "after all" hook:
     TypeError: Cannot read property 'kill' of null
      at Context.done (test/e2e/oauth2-authorization-co
```
Since, only policies loading was changed, so best guess was to try including some to fix those.
I thought that in the test in proxy `changeOrigin` is set to true, so `cors` need to be included, and others `jwt` because of `oauth2`. And no idea about `basic-auth`, but without it also, tests was breaking.

Reply:
Ok do not worry about these now, I can try to have a look into later.